resolwe_bio/docker_images/dnaseq/assets/amplicon_html_template.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+
+        <!-- Necessary bokeh includes: -->
+        <link href="bokeh-0.12.13.min.css" type="text/css"  rel="stylesheet" />
```

Comment:
You can remove the excessive whitespace between `type="text/css"` and `rel="stylesheet"`. The same goes for `type="text/javascript"` and `>` below (twice). Looks good otherwise.